content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -64,18 +64,72 @@ Terraform Cloud has support for Sentinel's modules feature. This allows you to w
 To configure a module, add a `module` entry to your `sentinel.hcl` file:
 
 ```hcl
-module "foo" {
-    source = "./modules/foo.sentinel"
+module "timezone" {
+    source = "./modules/timezone.sentinel"
 }
 ```
 
-The following entry would tell a policy check to load the code at `./modules/foo.sentinel` relative to the policy set working directory and make it available to be imported with a statement such as `import "foo"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
+Create a `modules` directory within the policy set working directory, and create a `timezone.sentinel` that looks as follows:
+
+```sentinel
+import "http"
+import "json"
+import "strings"
+
+uri = "https://raw.githubusercontent.com/dmfilipenko/timezones.json/master/timezones.json"
+request = http.get(uri)
+response = json.unmarshal(request.body)
+
+offset = func(abbr) {
+	timezone = filter response as _, r {
+		r.abbr is strings.to_upper(abbr)
+	}
+	for timezone as tz {
+		print("Getting timezone data for", tz.value)
+		print("Details:", tz.text)
+		print("Abbreviation:", tz.abbr)
+		print("Offset:", tz.offset)
+		return tz.offset
+	}
+}
+```
 
--> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/foo.sentinel` will not work.
+The above configuration would tell a policy check to load the code at `./modules/timezone.sentinel` relative to the policy set working directory and make it available to be imported with the statement `import "timezone"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
```

Comment:
This explanation seems to be attached to the HCL snippet above that makes the module available, and putting it after the Sentinel code example makes it a little harder to understand; on the other hand, moving it between them makes the section end too abruptly. 

I suggest splitting it; explain what the HCL block accomplishes next to the HCL block, then reiterate the `import` syntax down here.